src/styles/reset.scss

```diff
@@ -0,0 +1,6 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
```

Comment:
My advice is not to use a reset for all elements if you will style them. When recompiling, such a reset will increase the time of this process. This isn't considered bad practice, but it's something to be aware of.